File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/admob/FragmentBannerActivity.java
package com.jdroid.android.sample.ui.google.admob;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class FragmentBannerActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return FragmentBannerFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/admob/FragmentBannerActivity.kt
package com.jdroid.android.sample.ui.google.admob

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class FragmentBannerActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return FragmentBannerFragment::class.java
    }
}
